mirage/factories/preprint.ts

```diff
@@ -77,6 +78,8 @@ export default Factory.extend<PreprintMirageModel & PreprintTraits>({
     dateWithdrawn: null,
 
     doi: null,
+    preprintVersion: 1,
+    isLatestVersion: true,
```

Comment:
If this is the latest version, shouldn't it be version 3 based on the `withVersions` afterCreate?

Reply:
`withVersions` is a factory trait, so only preprints created with that trait (`server.create('preprint', 'withVersions')`) will have 3 versions. Every other preprint will just have only 1 version.